docs/blog/2018-1-25-building-a-site-with-react-and-contentful/index.md

```diff
@@ -0,0 +1,208 @@
+
+![Creating a content type](creating-content-type.png)
+
+![A list of events on the daily schedule](daily-schedule.png)
+
+![Creating an event for the daily schedule](daily-schedule.png)
+
+If you do create content directly in Contentful, make sure to name your fields in a way you can remember when you create GraphQL queries. If you use GraphiQL, it can suggest fields to you, but this will only help if the field names are clear and memorable.
+
+As far as pushing data out to my site goes, I didn't know which API or token I needed. I wondered why there were four APIs and two management tokens. The Contentful documentation on this was fairly fragmented, so I just installed the gatsby-source-contentful plugin and then put the example Space ID and Example token into my gatsby-config.js file. I think this was the right thing to do, because it worked.
+
+Gatsby-config.js file:
+
+```jsx
+module.exports = {
+  siteMetadata: {
+    title: `Taking Care of Watson`,
+  },
+  plugins: [
+    `gatsby-plugin-glamor`,
+    {
+      resolve: `gatsby-plugin-typography`,
+      options: {
+        pathToConfigModule: `src/utils/typography`,
+      },
+    },
+    //Next is where I include the gatsby-source-contentful plugin and include the spaceID and accessToken I got from my Contentful account.
+    {
+      resolve: `gatsby-source-contentful`,
+      options: {
+        spaceId: `g0b694a663ft`,
+        accessToken: `cc94019b03af3894ec87eded57fffd67c6785f1c6fb6b2c3e67ded27682b0c02`,
+      },
+    },
+    `gatsby-transformer-sharp`,
+    `gatsby-plugin-sharp`,
+  ],
+}
+```
+
+Once Contentful and Gatsby were getting along well together, I created pages and added a few other style-oriented plugins (Glamor and Typography) when I realized I wanted those. Then, I used GraphiQL to create queries that could pull my Contentful content into my site. After creating queries for each type of content I wanted (daily schedule, list of tricks & commands Watson knows, and a photo gallery), I saved each query in its corresponding page to see if anything broke. There were a few instances in which GraphiQL hadn’t let me know I had errors in the little ways I tried to sort data.
+
+Here’s an instance of a GraphiQL query that worked in there but not in the actual code. It needs to read `(sort: {field: [order]})` with the square brackets around the word "order" because there are multiple fields I can sort by; I had forgotten the brackets and GraphiQL said it would work, but then it didn't work. Here's the correct format:
+
+```jsx
```

Comment:
This should be `graphql` instead of `jsx`. Also could you copy the query to graphiql and click "pretty" so that it's formatted nicely?